nest-tax-backend/src/tasks/__tests__/tasks.service.spec.ts

```diff
@@ -47,115 +48,117 @@ describe('TasksService', () => {
   })
 
   describe('sendUnpaidTaxReminders', () => {
-    it('should not do anything when there are no taxes', async () => {
-      const findManyMock = jest
-        .spyOn(service['prismaService'].tax, 'findMany')
-        .mockResolvedValue([])
-      const trackEventUnpaidTaxReminderMock = jest.spyOn(
-        service['bloomreachService'],
-        'trackEventUnpaidTaxReminder',
-      )
+    if (process.env.FEATURE_TOGGLE_REMINDER_UNPAID_TAX) {
```

Comment:
This if is unnecessary if the `beforeEach` is set to true. If you set it to false and then implement this if, then you could've just commented out those tests altogether. Which in your use might also be understandable, however setting the env variable to true is much more indicative of what is happening without any further changes needed.